spec/index.bs

```diff
@@ -2282,6 +2282,46 @@ dictionary GPUImageBitmapCopyView {
 
   * {{GPUImageBitmapCopyView/origin}}: If unspecified, defaults to `[0, 0]`.
 
+### <dfn method for=GPUCommandEncoder>copyBufferToBuffer(source, sourceOffset, destination, destinationOffset, size)</dfn> ### {#GPUCommandEncoder-copyBufferToBuffer}
+
+<div algorithm="GPUCommandEncoder.copyBufferToBuffer">
+
+  **Arguments:**
+    - {{GPUBuffer}} |source|
+    - {{GPUSize64}} |sourceOffset|
+    - {{GPUBuffer}} |destination|
+    - {{GPUSize64}} |destinationOffset|
+    - {{GPUSize64}} |size|
+
+  **Returns:** void
+
+  Encode a command into the {{GPUCommandEncoder}} that copies |size| bytes of data from the |sourceOffset| of a {{GPUBuffer}} |source| to the |destinationOffset| of another {{GPUBuffer}} |destination|.
+
+</div>
+
+<div algorithm class=validusage>
+
+  <dfn abstract-op>copyBufferToBuffer Valid Usage</dfn>
+
+  Given a {{GPUCommandEncoder}} |encoder| and the arguments {{GPUBuffer}} |source|, {{GPUSize64}} |sourceOffset|, {{GPUBuffer}} |destination|, {{GPUSize64}} |destinationOffset|, {{GPUSize64}} |size|, the following validation rules apply:
+
+  - |encoder|.{{GPUCommandEncoder/copyBufferToBuffer()}} must be called before calling |encoder|.{{GPUCommandEncoder/finish()}}. // TODO: define the state machine for GPUCommandEncoder.
+  - |encoder|.{{GPUCommandEncoder/copyBufferToBuffer()}} must be called outside of any {{GPURenderPassEncoder}}s.
+  - |encoder|.{{GPUCommandEncoder/copyBufferToBuffer()}} must be called outside of any {{GPUComputePassEncoder}}s.
+  - |source| must be a [=valid=] {{GPUBuffer}}.
+  - |destination| must be a [=valid=] {{GPUBuffer}}.
+  - The {{GPUBuffer/[[usage]]}} of |source| must contain {{GPUBufferUsage/COPY_SRC}}.
+  - The {{GPUBuffer/[[usage]]}} of |destination| must contain {{GPUBufferUsage/COPY_DST}}.
+  - |size| must be a multiple of 4.
+  - |sourceOffset| must be a multiple of 4.
+  - |destinationOffset| must be a multiple of 4.
+  - (|sourceOffset| + |size|) must not overflow a {{GPUBuffer/[[size]]}}. // TODO: figure out how to handle overflows in the spec.
```

Comment:
Please use the Issue() syntax like at https://github.com/gpuweb/gpuweb/blob/master/spec/index.bs#L413 Same thing for the other TODO

Reply:
Done